CI/analysis/pvs_config.cfg

```diff
@@ -0,0 +1,26 @@
+exclude-path=*/blake2/*
+exclude-path=*/jansson/*
+exclude-path=*/json11/*
+exclude-path=*/liblzma/*
+exclude-path=*/lzma/*
+exclude-path=*/w32-pthreads/*
+exclude-path=*/plugins/enc-amf/*
+exclude-path=*/plugins/obs-browser/*
+exclude-path=*/plugins/obs-websocket/*
+exclude-path=*/plugins/win-dshow/libdshowcapture/*
+exclude-path=*/plugins/obs-outputs/ftl-sdk/*
+exclude-path=*/obs-build-dependencies/*
+exclude-path=*/simde/*
+exclude-path = *\blake2\*
```

Comment:
Duplication necessary because Windows is still basically DOS? ;)

Reply:
Not entirely sure yet, the PVS documentation is not great and things like possible config file options are split across multiple pages and none of them ever spell out how exactly it needs to be formatted. And since a single run takes ~80 minutes I didn't have time to test this yet. Will do later though.

Reply:
Seems to work with just unix-style paths as well. It's just odd that it isn't logged.